spec/models/notification_spec.rb

```diff
@@ -250,6 +250,32 @@ def process_alerts(post)
     end
   end
 
+  describe '.get_liked_by' do
+    let(:post) { Fabricate(:post) }
+    let(:user) { Fabricate(:user) }
+
+    before do
+      PostActionNotifier.enable
+    end
+
+    it 'should return the right notifications' do
+      expect(Notification.get_liked_by(user)).to eq([])
+
+      expect do
+        PostAlerter.post_created(Fabricate(:basic_reply,
+          user: user,
+          topic: post.topic
+        ))
+
+        PostAction.act(user, post, PostActionType.types[:like])
+      end.to change { Notification.count }.by(3)
```

Comment:
I don't care about the content of the other two notifications but I need to assert for the count to enforce that the test is filtering for a particular notification from 3 available ones. That is important because if we're filtering for a single notification when only that notification has been created, someone replacing the method with `Notification.all` will make the test pass.

Reply:
That makes sense, thanks.